packages/std/src/math/uint64.rs

```diff
@@ -63,6 +63,10 @@ impl Uint64 {
         self.0 == 0
     }
 
+    pub fn pow(&self, exp: u32) -> Self {
```

Comment:
Clippy probably does not like this reference because `self` can be copied.

Reply:
Oh, it's not. Can you derive Copy for Uint64 and check if clippy complains here?

Reply:
Clippy didn't seem to object

Reply:
Ah, okay. Maybe this only happens for other aruments, not the first one.